lib/lz4.c

```diff
@@ -333,7 +357,20 @@ static unsigned LZ4_NbCommonBytes (register size_t val)
             return (__builtin_clzll((U64)val) >> 3);
 #       else
             unsigned r;
+/* Disable warning "shift count is too large" */
```

Comment:
Classical issue.
This source code path is supposed to be only triggered in 64 bits mode.
So it will never generate any binary code on a 32-bits target.
Nonetheless, a compiler may complain, without realizing this is dead code.

We can probably get rid of it without these `#pragma`. A cast typically.